Tasks/Common/webdeployment-common/msdeployutility.ts

```diff
@@ -195,6 +195,9 @@ export function redirectMSDeployErrorToConsole() {
             else if(errorFileContent.indexOf("FILE_IN_USE") !== -1) {
                 tl.warning(tl.loc("Trytodeploywebappagainwithrenamefileoptionselected"));
             }
+            else if(errorFileContent.indexOf("Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.") != -1){
```

Comment:
Can we reduce the length of the hard-coded message?

Reply:
parsing just keyword now